src/operator-docs/transformation/concatMap.ts

```diff
@@ -1,6 +1,78 @@
 import { OperatorDoc } from '../operator.model';
 
 export const concatMap: OperatorDoc = {
-  'name': 'concatMap',
-  'operatorType': 'transformation'
+  name: 'concatMap',
+  operatorType: 'transformation',
+  signature: `concatMap(project: (value: T, index: number) =>  ObservableInput<I>,
+  ?resultSelector: (outerValue: T, innerValue: I, outerIndex: number, innerIndex: number) => R): Observable`,
+  parameters: [
+    {
+      name: 'project',
+      type: 'function(value: T, index: number): ObservableInput',
+      attribute: '',
+      description: `A function that, when applied to an item emitted by the source
+       Observable, returns an Observable.`
+    },
+    {
+      name: 'resultSelector',
+      type:
+        'function(outerValue: T, innerValue: I, outerIndex: number, innerIndex: number): any',
+      attribute: 'optional',
+      description: `A function to produce the value on the output Observable based on the values
+      and the indices of the source (outer) emission and the inner Observable
+      emission. The arguments passed to this function are:
+      - 'outerValue': the value that came from the source.
+      - 'innerValue': the value that came from the projected Observable.
+      - 'outerIndex': the "index" of the value that came from the source.
+      - 'innerIndex': the "index" of the value from the projected Observable.`
+    }
+  ],
+  marbleUrl: 'http://reactivex.io/rxjs/img/concatMap.png',
+  shortDescription: {
+    description: `Project each source value to an Observable which is merge in the output observable,
+    in a serialized fashion waiting for each one to complete before merging the next one
+    <span class="informal">Maps each value to an Observable, then flattens all of
+    these inner Observables using <a href="#/operators/concatAll">concatAll</a>.</span>`
+  },
```

Comment:
Just a few updates:

`Project each source value to an Observable which is then merged with the output observable in a serialized fashion. The previous subscription must complete before the next begins. Inner Observables are flattened using <a href="#/operators/concatAll">concatAll</a>.</span>`